rel8-jdk8-overrides.yaml

```diff
@@ -0,0 +1,41 @@
+schema_version: 1
+
+name: "jboss-eap-7/eap74-openjdk8-openshift-rhel8"
+
+labels:
+    - name: "com.redhat.component"
+      value: "jboss-eap-74-openjdk8-builder-openshift-rhel8-container"
+
+modules:
+      install:
+          - name: jboss.container.openjdk.jdk
+            version: "8"      
+          - name: jboss.container.eap.galleon.build-settings
+            version: "osbs"
+          - name: jboss.container.maven.module
+            version: "3.6"
+          - name: os-eap-python
+            version: '3.6'
+          - name: jboss.container.jolokia
+            version: '7'
+          - name: jboss.container.prometheus
+            version: '7'
+
+packages:
+  manager: dnf
+  content_sets_file: content_sets_rhel8.yml
+
+osbs:
+  configuration:
+    container:
+      platforms:
+        only:
+          - x86_64
+          #- s390x
```

Comment:
Is this correct and this image is not multiarch? Seems odd, since all of the others are.

Reply:
Yes confirmed with Mirek, only x86_64